### PR TITLE
[Merged by Bors] - feat: composition of localization functors

### PR DESCRIPTION
**Pull request by joelriou**
---
- [x] depends on: #6867
- [x] depends on: #6869
- [ ] depends on: #6887

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Localization/Composition.lean [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+namespace IsLocalization
+
+variable (L₁ W₁ L₂ W₂)
+
+lemma comp [L₁.IsLocalization W₁] [L₂.IsLocalization W₂]
+    (W₃ : MorphismProperty C₁) (hW₃ : W₃.IsInvertedBy (L₁ ⋙ L₂))
+    (hW₁₃ : W₁ ⊆ W₃) (hW₂₃ : W₂ ⊆ W₃.map L₁) :
+    (L₁ ⋙ L₂).IsLocalization W₃ := by
+  -- The proof proceed by reducing to the case of the constructed
```
Comment (jcommelin) [inline Mathlib/CategoryTheory/Localization/Composition.lean:60]:
```suggestion
  -- The proof proceeds by reducing to the case of the constructed
```

**Review by jcommelin — APPROVED** [review]
bors d+

**Comment by joelriou** [comment]
Thanks for the review!

bors merge

**Comment by joelriou** [comment]
bors merge

**Pull request closed without merging**